docs/CHANGELOG/changelog-0.9.x.md

```diff
@@ -120,4 +120,19 @@ To further harden Claudie, you may want to deploy our pre-defined network polici
  
 - Longhorn related issues, especially during node deletion resulted in many InputManifest issues, In this release we fixed the issues by switching to a different drain policy for longhorn replicas deployed across the nodes on the cluster, namely `block-for-eviction-if-last-replica`[#1596](https://github.com/berops/claudie/pull/1596) which results in:
     - Protecting data by preventing the drain operation from completing until there is a healthy replica available for each volume available on another node.
-    - Automatically evicts replicas, so the user does not need to do it manually.
\ No newline at end of file
+    - Automatically evicts replicas, so the user does not need to do it manually.
+
+ 
+## v0.9.2
+
+## What's Changed
+* Node local dns will be deployed on all newly build clusters [#1603](https://github.com/berops/claudie/pull/1603).
+  For existing clusters that were build using older Claudie version, this change will deploy the `node-local-dns` into the cluster
+  but it will not automatically work. Manual work needs to done by first editing the `kubelet-config` map in the `kube-system` namespace of the cluster
```

Comment:
```suggestion
  but it will not automatically work. Manual work needs to done by first editing the `kubelet-config` ConfigMap in the `kube-system` namespace of the cluster
```